File: sujeong/week9/H-Index.swift
import Foundation

func solution(_ citations:[Int]) -> Int {
    
    let sortedCitations:[Int] = citations.sorted().reversed()
        
    for (i, cit) in sortedCitations.enumerated() {
        if i >= cit {
            return i
        }
    }
    
    return citations.count
}

File: seungmi/week9/H-Index.py
def solution(citations):
    answer=0
    #모든 논문의 인용횟수가 0인경우
    if citations==[0]*len(citations):
        return 0
    for i in range(len(citations)+1):
        count=0
        for c in citations:
            if i<=c :
                count+=1
            if count>=i:
                answer=count
                break
    return answer